confluent_platform/assets/configuration/spec.yaml

```
@@ -25,3 +25,13 @@ files:
     options:
     - template: instances/jmx
     - template: instances/default
+  - template: logs
+    example:
+    - type: file
+      path: <CONFLUENT_COMPONENT_PATH>/logs/*.log
+      source: confluent_platform
+      service: <SERVICE_NAME>
+      log_processing_rules:
+      - type: multi_line
+        name: new_log_start_with_date
+        pattern: \[\d{4}\-\d{2}\-\d{2}
```

Should enclose this with single quote ?

```suggestion
        pattern: '\[\d{4}\-\d{2}\-\d{2}'
```

---

Seems not mandatory for this case.